ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/platform.py

```diff
@@ -95,6 +95,14 @@
             "non_mlag": 1020,
         },
     },
+    {
```

Comment:
Please check the various options under `feature_support` for platform settings and anything not following the defaults. At least I would expect `queue_monitor_length_notify: false` is needed (which should also be the case for all other X-series but is apparently not done)

Reply:
Please also update the schema which will reflect in docs.

Reply:
Will verify later today. Interestingly `bgp_update_wait_install` is not currently supported, so good point 👍

Reply:
@colinmacgiolla you still need to update the default value in the schema so it will be reflected in the docs.